orbit/pkg/packaging/macos_rcodesign.go

```diff
@@ -0,0 +1,67 @@
+package packaging
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+func RSign(pkgPath, cert string) error {
+	certPwd, ok := os.LookupEnv("MACOS_DEVID_CERTIFICATE_PASSWORD")
```

Comment:
Makes sense for this to be an argument and set the env var one level up, right?

Reply:
(Same with `AC_API_ISSUER` and `AC_API_KEY `)

Reply:
here I was also mimicking what we do in `NotarizeStaple` for native notarization. Will change.